grafana/README.md

```diff
@@ -0,0 +1,42 @@
+# Prometheus & Grafana for Nessie
+
+## Dev installation for local experiments
+
+These steps are mainly for setting up Prometheus/Grafana/Nessie locally.
+
+* Install Minikube as described in https://minikube.sigs.k8s.io/docs/start/
+* Install Helm as described in https://helm.sh/docs/intro/install/
+* Start Minikube cluster: `minikube start`
+* Create K8s Namespace: `kubectl create namespace nessie-dev`
```

Comment:
maybe all these steps for minikube and nessie could be avoided by just linking to https://github.com/projectnessie/nessie/blob/main/helm/README.md#dev-installation ?